rabbitmq-server-3.13.yaml

```diff
@@ -21,8 +21,8 @@ environment:
       - build-base
       - busybox
       - ca-certificates-bundle
-      # rabbitmq currently depends on v1.15 of elixir. Later versions will cause complication issues.
-      - elixir-1.15
+      - elixir
```

Comment:
Can you add a commit message comment or a comment here on why it is now OK to not pin the elixir version?

Reply:
We have moved the elixir-1.15 to enterprise-packages. We have left a copy here only for these package as with this approach we can build with the latest elixir-1.17. We can finally remove it from here.

Reply:
And also with elixir-1.15 it doesn't build [log with elixir-1.15](https://github.com/wolfi-dev/os/actions/runs/10068024953/job/27832635045#step:8:443)